index.html

```diff
@@ -84,7 +84,7 @@ <h2 class="heading">Sponsors &amp; backers</h2>
                 </div>
                 <div class="col-md-12 text-center">
                     <p>
-                        If you find JHipster useful for work, please consider asking your company to support this Open Source project by becoming <a href="https://opencollective.com/generator-jhipster#sponsor" target="_blank">a sponsor</a>. You can also individually sponsor the project by becoming <a href="https://opencollective.com/generator-jhipster#backer" target="_blank"> a backer</a>
+                        If you find JHipster useful for work, please consider asking your company to support this Open Source project by becoming <a href="https://opencollective.com/generator-jhipster#sponsor" target="_blank">a sponsor</a>. You can also individually sponsor the project by becoming <a href="https://opencollective.com/generator-jhipster#backer" target="_blank"> a backer</a>.
```

Comment:
I think you added a comma here by mistake

Reply:
I added a period at the end. It's currently missing on the site.

Reply:
Oh sorry, I must check my glasses!!